app/controllers/lessons_controller.rb

```diff
@@ -9,9 +9,8 @@ def show
   private
 
   def set_user
-    if user_signed_in?
-      @user = User.includes(:completed_lessons).find(current_user.id)
-    end
+    return unless user_signed_in?
```

Comment:
can we change this back to the way it was please? It reads better with an if imo.

Reply:
Done.